### PR TITLE
UI improvements for ColorSwitch type devices

### PR DESCRIPTION
**Pull request by emontnemery**
- Show dimmer slider on dashboard and lights view
- Show "Brightness Up" / "Brightness Down" buttons instead of master level slider for MiLight v4/v5 devices that don't support absolute dimming
- Show "Warmer" / "Colder" buttons instead of color temeperature picker for MiLight v4/v5 devices that don't support absolute setting of color temperature

### CODE REVIEW

File: www/app/DeviceLightEdit.js [hunk 1]
```diff
@@ -61,14 +61,17 @@ define(['app'], function (app) {
             var $ctrl = this;
 
             $ctrl.$onInit = init;
-            $ctrl.isBrightnessAvailable = isBrightnessAvailable;
+            $ctrl.isRelativeDimmer = isRelativeDimmer;
+            $ctrl.isRelativeTemperature = isRelativeTemperature;
```
Comment (stas-demydiuk) [inline www/app/DeviceLightEdit.js]: What do you think about renaming it into `isRealtiveColorTemperature`?

File: www/app/DeviceLightEdit.js [hunk 2]
```diff
@@ -93,20 +97,29 @@ define(['app'], function (app) {
...
+            function isRelativeTemperature() {
                 var ledType = getLEDType($ctrl.device.SubType);
-                var isColorSwitch = $ctrl.device.Type === 'Color Switch';
 
-                return ledType.bHasRGB && $ctrl.device.Unit === 0 && isColorSwitch
+                return $ctrl.device.DimmerType && $ctrl.device.DimmerType === "rel" && ledType.bHasTemperature;
             }
 
+            // returns true if the light supports extended functions such as nightlight and fulllight
             function isLightControlAvailable() {
```
Comment (stas-demydiuk) [inline www/app/DeviceLightEdit.js]: Maybe we can get rid of this function as it always return `true`?

**Comment by emontnemery** [comment]
Updated:
- Rebased to include #2316 
- Fix review comments
- Remove app/DeviceTimers.js from appcache

**Comment by gizmocuz** [comment]
Thanks!

**Pull request merged**